serverless.yml

```diff
@@ -1,4 +1,4 @@
-service: aws-python-scheduled-cron
+service: scheduled-cron-error-handling
```

Comment:
Maybe leave this unchanged until we pick a project name for the template?

Reply:
This was to avoid overwrites of other peoples deployment/testing